sunkit_image/tests/test_offlimb_enhance.py

```diff
@@ -1,12 +1,55 @@
-import os
-
 import numpy as np
 import pytest
 import astropy.units as u
-from numpy.testing import assert_allclose
 
 import sunpy
 import sunpy.map
 import sunpy.data.test
+import sunkit_image.utils.utils as utils
+import sunkit_image.offlimb_enhance as off
+
+
+test_map_data = np.ones((5, 5))
+header = {}
+test_map = sunpy.map.Map((test_map_data, header))
+radial_bin_edges = utils.equally_spaced_bins()
+radial_bin_edges = radial_bin_edges*u.R_sun
```

Comment:
Could these be made a `pytest.fixture`?

Reply:
Also what happens if we create a test image with one circle ring of different values?

Reply:
Pytest fixtures made.
Even if we create an image of one circular ring with different values it would be difficult to manually calculate what would be the intensity values once the filter is applied.

Reply:
Sure but we should look into. We will need more numerical tests.